src/App.tsx

```diff
@@ -1,5 +1,39 @@
 import React, { ReactElement } from 'react';
+import ListPage from './ListItem';
+
+const personArray = [
```

Comment:
test data must be removed eventually and replaced with a function to fetch backend data